### ISSUE TITLE
Safari pinned tab icon color

### ISSUE THREAD

**Issue by Arttse** [validator/validator#426]
Interested in resolving some question.

In the documentation [Apple Creating Pinned Tab Icons](https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html) provides a `color` attribute in the `link` element.

With this code example:
```html
<!DOCTYPE html>
<html>
    <head>
        <title>Test</title>
        <link rel="mask-icon" href="website_icon.svg" color="red">
    </head>
    <body>
        <p></p>
    </body>
</html>
```
Validator returns `Error: Attribute color not allowed on element link at this point`.

Is it possible something to do with it? Thank you!

**Comment by sideshowbarker** [comment]
duplicate of https://github.com/validator/validator/issues/174 I think?

…which is blocked on https://github.com/whatwg/html/issues/110

**Comment by Arttse** [comment]
Yes, similar to. Sorry, bad looking, my fault.

Sadly...
We will wait for at least one of the parties will adopt the `color` attribute or come up with something else. And in the meantime will have to settle for an inappropriate error.

**Comment by sideshowbarker** [comment]
closing per https://github.com/validator/validator/issues/426#issuecomment-266697899
